packages/opentelemetry-exporter-prometheus/src/prometheus.ts

```diff
@@ -59,7 +58,7 @@ export class PrometheusExporter implements MetricExporter {
    * @param config Exporter configuration
    * @param callback Callback to be called after a server was started
    */
-  constructor(config: ExporterConfig = {}, callback?: () => void) {
+  constructor(config: ExporterConfig, callback?: () => void) {
```

Comment:
I think this should not be changed, if in js user will not define any config we don't want the error "object is undefined" to be thrown.

Reply:
@legendecas any update here?